src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -49,15 +49,12 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
       String zip = request.getParameter("zipCode");
       smallCityService.createUserWithZip(zip);
       LOGGER.warning(e.getMessage() 
-            + "Unable to geolocate user, zipCode entered instead.");
+           + "Unable to geolocate user, zipCode entered instead.");
     }
-  }
-
-  @Override
-  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+  
     response.setContentType("application/json;");
     response.setCharacterEncoding("UTF-8");
-    response.getWriter().println(convertToJson(smallCityService.getBusinesses()));
+    response.getWriter().println(convertToJson(smallCityService.getSmallBusinesses()));
```

Comment:
Can this method be removed altogether? It's only being used in the test now, which should also be updated.

Reply:
Yes the method should be removed -- @AbakirH is there a way to do your test without this function?

Reply:
I created an issue to track this #50

Reply:
I can remove it, but until this pull request is approved.

Reply:
Can we remove it in this pull request?

Reply:
Never mind I do not think we can remove this function, as the bigbusinessesTest java file, we do not want to test the places API within the first test. Thus, by calling getSmallBusinesses, it overrides my local list and I only want to test the filiterBySmallBusinesses method with the local list in the test. 

A work around to this problem is to make the variable businesses in smallCItyService public so it can be accessed anywhere.

Reply:
Added a comment describing the reason for this function

Reply:
Was this added? I'm not seeing it.

Reply:
It's on line 73-74 of SmallCityService.java